user/models.py

```diff
@@ -1,5 +1,48 @@
 from django.contrib.auth.models import AbstractUser
```

Comment:
```suggestion
from django.contrib.auth.models import AbstractUser, BaseUserManager
```
and you can delete import below